jobs/service-fabrik-broker/templates/config/keepalived/master.sh.erb

```diff
@@ -0,0 +1,27 @@
+iaas_name=<%= p('backup.provider.name') %>
```

Comment:
i think we must keep HA feature independent of backup and hence different attributes in manifest to provide IAAS info.